eng/nunit.runsettings

```diff
@@ -5,8 +5,12 @@
     </NUnit>
     <RunConfiguration>
         <TargetPlatform>x64</TargetPlatform>
+
+        <!-- Return non-zero exit code if a test run finds no tests -->
+        <!-- disable for now because there are a lot of test projects that don't find tests and would start failing -->
```

Comment:
Issue https://github.com/Azure/azure-sdk-for-net/issues/29139 filed to enable this.